src/AbstractAlgebra.jl

```diff
@@ -377,7 +377,7 @@ import .Generic: add!, addeq!, addmul!, add_column, add_column!, add_row,
                  isdomain_type, isexact_type, isgen, ishessenberg,
                  ishnf, ishomogeneous, isisomorphic, ismonomial,
                  isone, isreverse, isrimhook,
-                 isrref, issquare, issubmodule, isterm,
+                 isrref, issquare, issubmodule, issymmetric, isterm,
```

Comment:
Seems to be the case. But then the same should apply to `det`, `hessenberg!`, and `rank` (I followed the example set by `det`).

Reply:
Yeah, these are imported from LinearAlgebra in both AbstractAlgebra and Generic (which is correct). But this means the Generic version overloads the LinearAlgebra one.

So you could try removing those other ones too (det, rank, etc.) if you want. So long as they are imported in both AbstractAlgebra and Generic, everything should keep working (hopefully).

Reply:
If that seems to work, you could put a note above the LinearAlgebra exports to the effect:

# If you want to add methods to functions in Base or LinearAlgebra they should be imported here and in Generic.jl as well, # and not imported/exported anywhere else.

Reply:
So, it seems that not importing the functions from `.Generic` works fine. But they have to be exported, otherwise they throw `UndefVarError` (if you don't do `using LinearAlgebra`).

For example, `transpose!` and `norm` get imported, but not exported (maybe they are only used internally?).

Concerning the note, is there a consensus on exporting `Base` methods? Some are exported (e.g. `size`), some are not (e.g. `abs`). Could this be relevant for people using `baremodule` without `Base`?

Reply:
Ah yes, of course LinearAlgebra is different to Base in that regard.

As for exporting Base methods, I guess there's no need.